srcpkgs/rogue/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'rogue'
+pkgname=rogue
+version=5.4.4
+revision=1
+archs="x86_64-glibc"
+makedepends=ncurses-devel
+depends="ncurses"
+short_desc="Dungeon crawl game from the 80s"
+maintainer="sirkhancision <jsantiago12tone@gmail.com>"
+license="custom:Rogue"
+homepage="http://rogue.rogueforge.net/"
+distfiles="https://github.com/phs/rogue/archive/v$version.tar.gz"
+checksum=aea2204f046576e06ba1bc53808cc193306e4a694a92a573e739289117f91a41
+
+do_build() {
+	./configure
+	make
+}
```

Comment:
```suggestion
```
not needed with the build_style, and configure should be in do_configure() if it's explicit anyways